utils/functions

```diff
@@ -145,7 +145,23 @@ m0_local_nid_get()
         local iface=`cat /etc/libfab.conf | grep '\S' | grep --invert-match "^ *#" | cut -d "(" -f2 | cut -d ")" -f1`
         local proto=`cat /etc/libfab.conf | grep '\S' | grep --invert-match "^ *#" | cut -d "=" -f2 | cut -d "(" -f1`
         local ip_addr=`ip addr show $iface |grep inet| head -1 | awk '{print $2}'|cut -d '/' -f1`
-        local_nid=$ip_addr@$proto
+        # Currently using ipv4 address (inet)
+        local_nid=inet:$proto:$ip_addr
+    fi
+    echo $local_nid
```

Comment:
I think this is for debug and you need to remove this line once the debugging is complete.

Reply:
No, It is not for debugging. When we call `m0_local_nid_get()` it prints out the value of `local_nid` and that value gets assigned to the variable.